vernac/vernacentries.ml

```diff
@@ -675,9 +672,9 @@ let extract_inductive_udecl (indl:(inductive_expr * decl_notation list) list) =
     indicates whether the type is inductive, co-inductive or
     neither. *)
 let vernac_inductive ~atts cum lo finite indl =
-  let open DefAttributes in
   let atts, template = Attributes.(parse_with_extra template atts) in
-  let atts = parse atts in
+  let extra, poly = Attributes.(parse_with_extra polymorphic atts) in
+  unsupported_attributes extra;
```

Comment:
Just do `let poly, template = Attributes.(parse Notations.(polymorphic ++ template) atts) in`

Reply:
Done, thanks!